app/main/lib/presto.py

```diff
@@ -26,6 +26,7 @@ def get_similar_items_callback_url(alegre_host, similarity_type):
     def send_request(presto_host, model_key, callback_url, message, requires_callback=True):
         data = {
             "callback_url": callback_url,
+            "content_hash": message.get("content_hash", str(uuid.uuid4())),
```

Comment:
I can see we definitely need a default random value if no hash provided so that it won't cache.  But in this case it isn't a hash.  I'm wondering if we need to know the type of hash used to know if we can compare hashes?  Or I guess non-comparable hashes will always result in non-matches so doesn't matter since the use case is caching?

Reply:
Technically we don't need this, according to the spec on the presto side - given that I think I want to just remove the default value altogether!